bigframes/dataframe.py

```diff
@@ -2548,34 +2561,24 @@ def unstack(self, level: LevelsType = -1):
         )
         return DataFrame(pivot_block)
 
+    def _filter_columns_by_type(self, types: Set[bigframes.dtypes.Dtype]) -> DataFrame:
```

Comment:
Rather than create a private function for this, let's instead implement [select_dtypes](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.select_dtypes.html) and use that.

Reply:
It turned out that select_dtypes() is already implemented in BF. Code updated.